wasp/wasp.py

```diff
@@ -504,4 +507,27 @@ def schedule(self, enable=True):
 
         self._scheduling = enable
 
+
+    def set_theme(self, new_theme: bytes):
+        self._theme = array.array('H', new_theme)
+    def theme(self, theme_part: str) -> int:
+        """Returns the relevant part of theme. For more see ../tools/themer.py"""
+        theme_parts = ("ble",
+                       "scroll-indicator-up",
+                       "scroll-indicator-down",
+                       "battery-charging",
+                       "battery-empty",
+                       "battery-full",
+                       "status-clock",
+                       "confirm-yes",
+                       "confirm-no",
```

Comment:
I don't think confirm-yes/no should be candidates for themeing.

Reply:
I disagree - a red no button and a green yes button is an example I could come up with

Reply:
note that GNOME 3 also has colored buttons sometimes.

Reply:
![image](https://user-images.githubusercontent.com/30871100/101284143-328ce680-37e7-11eb-96cb-b59265e852c7.png)
![image](https://user-images.githubusercontent.com/30871100/101284174-59e3b380-37e7-11eb-8027-80428a3d2d59.png)

see images for examples ^

Reply:
>> I don't think confirm-yes/no should be candidates for themeing.
>
> I disagree - a red no button and a green yes button is an example I could come up with

That's an argument for colouring the buttons! It doesn't have anything to do with theming. Note that the current YES/NO icons don't match the visual style of the rest of wasp-os so I'm already working on an update to the confirmation view that will change the artwork and add colour to the buttons.

However adding something to a colour palette theme system like the one you're working on implies that is makes sense to draw the UI component in a different colour and that doing so will make the user feel happier when they glance at their watch.

I simply don't think the colour of No/Yes buttons are good candidates for theming. There are only a narrow set of colour themes that make sense and users seldom see the dialog anyway. IMHO the best way to make the user(s) happy when they do see these buttons is to make them more attractive rather than the make the current ones themeable.

Reply:
okay, will remove the confirmation dialog from the theming engine.